include/RED4ext/Scripting/Natives/Generated/anim/AnimSet.hpp

```diff
@@ -4,18 +4,16 @@
 
 // This file is generated from the Game's Reflection data
 
-#include <cstdint>
-#include <RED4ext/Common.hpp>
-#include <RED4ext/DynArray.hpp>
-#include <RED4ext/Handle.hpp>
-#include <RED4ext/NativeTypes.hpp>
-#include <RED4ext/Scripting/Natives/Generated/CResource.hpp>
-#include <RED4ext/Scripting/Natives/Generated/anim/AnimDataChunk.hpp>
-#include <RED4ext/Scripting/Natives/Generated/anim/AnimFallbackFrameDesc.hpp>
-#include <RED4ext/Scripting/Natives/Generated/red/TagList.hpp>
+#include <RED4ext/Scripting/Natives/AnimSet.hpp>
 
 namespace RED4ext
 {
+RED4EXT_ASSERT_SIZE(AnimSet, 0x1F0);
+using animAnimSet = anim::AnimSet;
+} // namespace RED4ext
+
+/*namespace RED4ext
```

Comment:
Can you regenerate the dumps and copy them here?

Reply:
I'm not sure what you mean

Reply:
I would like to have only generated files under `Generated` directory. So if you can run this plugin https://github.com/WopsS/RED4.RTTIDumper or any other plugin by calling `RED4ext::GameReflection::Dump(PATH);` and then copy these files here would be great.

This way we make sure that the generated files are OK, even when we run the dumper in the future.

Reply:
Does that work?

Reply:
Are these comments added by you or by the generator? If they are added by you then something is not configured correctly.

Reply:
I added them cause I had forgotten I had remapped the classes and that this wasn't necessary. Let me know if I need to make more changes